stanford_video.views_default.inc

```diff
@@ -152,13 +152,6 @@ function stanford_video_views_default_views() {
   $handler->display->display_options['fields']['views_bulk_operations']['vbo_settings']['force_single'] = 0;
   $handler->display->display_options['fields']['views_bulk_operations']['vbo_settings']['entity_load_capacity'] = '10';
   $handler->display->display_options['fields']['views_bulk_operations']['vbo_operations'] = array(
-    'action::node_assign_owner_action' => array(
```

Comment:
OK, let me figure out why it installs as overridden and can't be reverted, then.